packages/core/src/managed/overview/Resource.tsx

```diff
@@ -63,7 +63,7 @@ const Status = ({
         />
         <div>
           <div>{state.reason || statusUtils[state.status].defaultReason}</div>
-          {state.event && state.event !== state.reason && <div>{state.event}</div>}
+          {state.event && state.event !== state.reason && <Markdown className="event" message={state.ev} />}
```

Comment:
This line is causing the build failures: 

```js
src/managed/overview/Resource.tsx:66:102 - error TS2339: Property 'ev' does not exist on type '{ __typename?: "MdResourceActuationState"; } & Pick<MdResourceActuationState, "status" | "reason" | "event"> & { tasks?: ({ __typename?: "MdResourceTask"; } & Pick<...>)[]; }'.

66           {state.event && state.event !== state.reason && <Markdown className="event" message={state.ev} />}
```

Reply:
Thanks. I probably deleted part of it accidentally